### ISSUE TITLE
Documentation on readthedocs.io

### ISSUE THREAD

**Issue by Lemmih** [noinia/hgeometry#42]
readthedocs.io hosts documentation written in markdown for free. I think it would be a good fit for documentation that goes beyond haddock.

@noinia I can configure it with github hooks if you add me as a collaborator.

**Comment by noinia** [comment]
I don't think we really have such documentation at this point right? Or do you have a particular "part" of hgeometry in mind that specifically needs this kind of documentation? Clearly, having more documentation is certainly welcome, but my preference would be to have it as tightly integrated with the package and its builds as possible, so to make sure things don't go out of sync.

**Comment by Lemmih** [comment]
We don't have any such documentation right now but I'd like to write some. Tutorials and how-to guides could be part of the haddock docs but I would prefer to keep them separate, especially if they are full of pictures.
